ci-operator/step-registry/baremetal/lab/ipi/conf/external-lb/baremetal-lab-ipi-conf-external-lb-commands.sh

```diff
@@ -14,11 +14,11 @@ trap 'CHILDREN=$(jobs -p); if test -n "${CHILDREN}"; then kill ${CHILDREN} && wa
 
 echo "Creating patch file to enable UserManaged loadbalancer: ${SHARED_DIR}/install-config.yaml"
 
-cat > "${SHARED_DIR}/external_lb_patch_install_config.yaml" <<EOF
+cat > "${SHARED_DIR}/external_lb_append.patch_install_config.yaml" <<EOF
 networking:
   machineNetwork:
-  - cidr: ${INTERNAL_NET_CIDR}
-  - cidr: $(yq ".api_vip" "${SHARED_DIR}/external_vips.yaml")/32
+  $([ "${ipv4_enabled:-false}" == "true" ] && echo "- cidr: $(yq ".api_vip" "${SHARED_DIR}/external_vips.yaml")/32")
```

Comment:
Do you mind only adding a comment about the bug that is causing us having these lines?
something like:
```
TODO: Remove when https://issues.redhat.com/browse/OCPBUGS-35811 is fixed
```